website/content/en/docs/scorecard/custom-tests.md

```diff
@@ -0,0 +1,185 @@
+# Custom Tests using Operator SDK Scorecard
```

Comment:
Should this doc appear on the website? if yes, then is required to put the header as it has in the other files.

Reply:
I don't think this should appear still. But I will check if Jeff regarding this.